packages/shared.test/src/handlers/util/handleOpenApiCall.ts

```diff
@@ -94,30 +91,39 @@ export function handleOpenApiCall<
     responseMediaType?: string;
   } = {} as any;
 
-  const capture: OpenApiRequest<any> = (
-    method,
-    endPoint,
-  ): Promise<any> => {
-    captured = {
-      method: method as any,
-      endPoint,
-    };
+  const capture = {
+    fetch: (url: string, req: Request) => {
+      const u = new URL(url);
+      u.search = ""; // msw doesn't want the search string
+      captured = {
+        method: req.method as any,
+        endPoint: u.toString().replace(/%3A/g, ":"),
+      };
 
-    return Promise.resolve();
+      // fake a response object so the call doesn't fail
+      return {
+        ok: true,
+        json: () => Promise.resolve({}),
+        blob: () => new Blob(),
+      };
+    },
+    baseUrl: "https://stack.palantir.com/",
   };
 
   // we don't care about the promise here, we are just building the url
   openApiCall(
     capture as any,
     ...(names.map(n => `:${n}`) as any),
+    // add a simulated blob in here in case of an upload
+    { type: "", size: 5 },
   );
 
   return http
     [captured.method.toLowerCase() as Lowercase<typeof captured.method>](
-      `https://stack.palantir.com/api${captured.endPoint}`,
+      captured.endPoint,
       authHandlerMiddleware(async (info) => {
         try {
-          const result = await restImpl(
+          const result: any = await restImpl(
```

Comment:
Why do we need this?

Reply:
The line on 130 produces an error: `The left-hand side of an 'instanceof' expression must be of type 'any', an object type or a type parameter.ts(2358)` without it.